heat/core/devices.py

```diff
@@ -132,3 +133,52 @@ def use_device(device=None):
     """
     global __default_device
     __default_device = sanitize_device(device)
+
+
+def _use_envar_device():
```

Comment:
make sure that you add this function to `__all__` so it can be accessed outside of the import.

Reply:
It's a secret to everybody :slightly_smiling_face: